packages/formstr-sdk/package.json

```diff
@@ -19,6 +19,7 @@
     "dist/**"
   ],
   "scripts": {
+    "prebuild": "rm -rf ./dist",
```

Comment:
looks dangerous why do we need this?

Reply:
although running multiple workspace builds in parallel/concurrently might fail

Reply:
removed